huaweicloud/services/dws/data_source_huaweicloud_dws_flavors.go

```diff
@@ -215,6 +222,17 @@ func flattenListNodeTypesFlavors(resp interface{}) []interface{} {
 	return rst
 }
 
+func obtainFlavorsDatastoreVersion(resp interface{}) string {
```

Comment:
suggest to use `flattenXXXX`
```suggestion
func flattenFlavorsDatastoreVersion(resp interface{}) string {
```